src/cmds/openapi/index.ts

```diff
@@ -34,7 +34,7 @@ export default class OpenAPICommand extends Command {
     super();
 
     this.command = 'openapi';
-    this.usage = 'openapi [file] [options]';
+    this.usage = 'openapi [file|url] [options]';
```

Comment:
[oas-normalize](https://npm.im/oas-normalize), which all of these commands use for spec processing, handles URLs so we should document in these commands that you can supply them either of these.